doc-content/enterprise-only/employee-rostering/optashift-ER-openshift-deploy-con.adoc

```diff
@@ -1,8 +1,9 @@
 [id='optashift-ER-openshift-deploy-con']
 = Installing and starting the employee rostering starter application on OpenShift
 
-You can deploy the Employee Rostering starter application to {OPENSHIFT} using an OpenShift template or using the `runOnOpenShift.sh` shell script that is provided in the *{PRODUCT} {PRODUCT_VERSION_LONG} Kogito and OptaPlanner 8 Decision Services Quickstarts* distribution.
+You can use an OpenShift template or the `runOnOpenShift.sh` script to deploy the Employee Rostering starter application to {OPENSHIFT}. The  `runOnOpenShift.sh` shell script is available in the *{PRODUCT} {PRODUCT_VERSION_LONG} Kogito and OptaPlanner 8 Decision Services Quickstarts* distribution.
 
-If you use an OpenShift template, {OPENSHIFT} loads an image with the application from the Red Hat repository and deploys it. This method of deployment is a technological preview.
+If you use an OpenShift template, {OPENSHIFT} loads an image with the application from the Red Hat repository and deploys it. This method of deployment is a technology preview feature.
+// Christopher, is this still technology preview?
 
-If you use the script, it builds and packages the application source code locally and uploads it to the OpenShift environment for deployment.  You can use this method when Java developer tools (Java Development Kit and Maven) and a bash shell command line are available.
+The `runOnOpenShift.sh` script builds and packages the application source code locally and uploads it to the OpenShift environment for deployment.  This method requires Java Development Kit, Apache Maven, and a bash shell command line.
```

Comment:
```suggestion
The `runOnOpenShift.sh` script builds and packages the application source code locally and uploads it to the OpenShift environment for deployment.  This method requires the Java Development Kit, Apache Maven, and a bash shell command line.
```

Reply:
I disagree with this suggestion. Java Development Kit is a proper noun so no article before. It's a bit confusing because of 'kit' but for example, we don't say the Apache Maven, or the KIE Server (or we shouldn't - i know it slips through sometimes).

Reply:
Yep. the "kit" got me all right. Grand so.